data_steward/utils/bq.py

```diff
@@ -1,8 +1,7 @@
 from google.cloud import bigquery
 
-client = bigquery.Client()
-
 
 def query(q, use_cache=False):
+    client = bigquery.Client()
```

Comment:
I would suggest changing this to something like `client = get_client(<project_id>)` by defining a `get_client(project_id=None)` function definition.

Reply:
sure will do.